package.json

```diff
@@ -14,14 +14,12 @@
     "dist-ci": "npm-run-all dist:* dist:*:*",
     "dist:gulp": "gulp",
     "dist:docs": "gulp docs",
-    "dist:prettier": "prettier ./{dist,docs}/*.{scss,css,json,js} --write",
-    "dist:colors:prettier-sketchpalette": "prettier ./dist/*.sketchpalette --write --parser json",
+    "dist:format": "sewing-kit format",
+    "dist:colors:format-sketchpalette": "prettier ./dist/*.sketchpalette --write --parser json",
```

Comment:
I think keep the overrides section on the prettierrc, then you can remove the `--parser` here.

Reply:
Good point!